src/commands/init/init.py

```diff
@@ -1,73 +1,46 @@
-from collections import OrderedDict
 from pathlib import Path
 import shutil
-
-import tomli_w
 from colorama import Fore
 
+from src.utils.config.package import Package, PackageConfig
+
 
 def init(script_root: str):
     """
     Creates init protostar project
     """
     project_name = input(f"{Fore.CYAN}Project name: ")
 
-    project_path = Path() / project_name
-    copy_template(script_root, "default", project_path)
-
-    package = PackageConfig(project_path=project_path)
+    project_root = Path() / project_name
+    copy_template(script_root, "default", project_root)
+    package = Package(project_root=project_root)
 
-    package.name = project_name
+    def colored_input(message: str):
+        return input(f"{Fore.CYAN}{message}:{Fore.RESET} ")
```

Comment:
Please use `src.utils.log_color_provider`. Sometimes a user may want to disable colors, for example when running a script on the CI.